src/cljs/athens/devcards/node_page.cljs

```diff
@@ -149,30 +196,28 @@
 
    ;; References
    (for [[linked-or-unlinked refs] ref-groups]
-     [:div {:key linked-or-unlinked}
-      [:div (use-style {:display         "flex"
-                        :justify-content "space-between"
-                        :align-items "center"})
-       [:h3 linked-or-unlinked]
-       [:span
-        [button {:label    [(r/adapt-react-class mui-icons/FilterList)]
-                 :disabled true}]]]
-      (doall
-        (for [[group-title group] refs]
-          [:<> {:key group-title}
-           [:h4 group-title]
-           (for [{:block/keys [uid parents] :as block} group]
-             [:div {:key uid}
-              ;; TODO: replace with breadcrumbs?
+     (when (> (count refs) 0)
+       [:section (use-style references-style {:key linked-or-unlinked})
+        [:h4 (use-style references-heading-style)
+         [(r/adapt-react-class mui-icons/Link)]
+         [:span linked-or-unlinked]
+         [button {:label    [(r/adapt-react-class mui-icons/FilterList)]
+                  :disabled true}]]
+        [:div (use-style references-list-style)
+         (doall
+           (for [[group-title group] refs]
+             [:div (use-style references-group-style {:key group-title})
+              [:h4 (use-style references-group-title-style)
+               [:a {:on-click #(navigate-uid uid)} group-title]]
```

Comment:
Roam behavior is to expand the reference without navigation. May implement this down the line.

Reply:
@tangjeff0 What does this mean? Does it add another reference item?

Reply:
@shanberg https://www.loom.com/share/e396caf6ccfa47de8400f067baf6f3cb